wallet-api/tutorial/Tutorial.md

```diff
@@ -157,7 +156,10 @@ Then we compute the total value of all transaction inputs, using `P.foldr` on th
 ```haskell
                   totalInputs :: Int
                   totalInputs =
+                      -- define a function "v" that extracts the ada value from a 'PendingTxIn'
                       let v (PendingTxIn _ _ (Value vl)) = vl in
+
+                      -- Apply "v" to each transaction input, summing up the results
                       $$(P.foldr) (\i total -> total + v i) 0 ins
```

Comment:
You could just do the pattern match in the lambda argument and then you wouldn't need the helper function.

Reply:
True, but I think this is slightly more readable.